app/src/main/java/com/hcodez/android/ui/MainMenu.java

```diff
@@ -0,0 +1,46 @@
+package com.hcodez.android.ui;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.SearchView;
+
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+import com.hcodez.android.R;
+
+public class MainMenu extends Activity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        FloatingActionButton buttonAdd = findViewById(R.id.buttonAdd);
```

Comment:
these buttons might need to be reused, declare them as a private field in the class and follow naming standards